tensorflow/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: tensorflow
+  version: "0.5.0"
+
+build:
+  number: 1
+  entry_points:
+    - tensorflow_model_cifar10_eval = tensorflow.models.image.cifar10.cifar10_eval:main
+    - tensorflow_model_cifar10_multi_gpu_train = tensorflow.models.image.cifar10.cifar10_multi_gpu_train:main
+    - tensorflow_model_cifar10_train = tensorflow.models.image.cifar10.cifar10_train:main
+    - tensorboard = tensorflow.tensorboard.tensorboard:main
+    - tensorflow_model_mnist_convolutional = tensorflow.models.image.mnist.convolutional:main
+
+requirements:
+  build:
+    - python
+    - pip
+    - numpy >=1.9.2
+    - six >=1.10.0
+  run:
+    - python
+    - numpy >=1.9.2
+    - six >=1.10.0
+
+test:
+  imports:
+    - tensorflow
+
+about:
+  home: http://tensorflow.com/
```

Comment:
Interesting, that is the URL that Google is putting in the Wheel metadata but it is not correct.  Should be http://tensorflow.org/.  I'll update momentary.